src/structures/Presence.js

```diff
@@ -32,9 +32,9 @@ class Presence {
     /**
      * The devices this presence is on
      * @type {?object}
```

Comment:
Could you also capitalize the `o` while you are at it?
```suggestion
      * @type {?Object}
```